app/outliers.py

```diff
@@ -191,7 +116,7 @@ def print_analysis_summary(analyzed_models):
                 should_schedule_next_run = True
 
             # Check for configuration file changes and load them in case it's needed
-            if len(file_mod_watcher.files_changed()) > 0:
+            if file_mod_watcher.files_changed():
```

Comment:
Here the condition will be done with a number (`files_changed` return the number of file that have changes)